crates/biome_deserialize_macros/src/lib.rs

```diff
@@ -79,14 +79,85 @@ use syn::{parse_macro_input, DeriveInput};
 /// }
 /// ```
 ///
+/// ### `from`
+///
+/// This attribute allows deserializing a type,
+/// and then converting it to the current aannotated type.
+/// The annotated type must implement the standard `From` trait.
+///
+/// For structs and enums that also implement Serde's `serde::Deserialize`,
+/// it automatically picks up on Serde's `from` attribute.
+/// `deserializable(from = _)` takes precdence over `serde(from = _)`.
+///
+/// ```no_test
+/// #[derive(Default, Deserializable)]
+/// #[deserializable(from = Deserialized)]
+/// struct Converted {
+///     fullname: String,
+/// }
+///
+/// #[derive(Default, Deserializable)]
+/// struct Deserialized {
```

Comment:
The name deserialized can be confusing (at least internally), because we also have a type called like this. We can pick another name, perhaps `Person`